### ISSUE TITLE
Ordering of layouts

### ISSUE THREAD

**Issue by SicariusNoctis** [RasmusLindroth/i3keys#8]
The ordering is currently based on which appears first in i3config. For me, this is:

```
Mod4+shift
Mod4
Mod4+Shift
No modifiers
Mod1+Mod4
Mod1
Mod1+shift
Mod4+ctrl
Mod4+ctrl+shift
Mod1+Mod4+shift
ctrl+shift
ctrl
shift
Mod1+Mod4+ctrl
```

It would be nice to get some consistency in sort order. For instance:

```
Mod4             # which I use more than Mod1... auto-detected?
Mod4+Shift       # combined with Mod4+shift
Mod4+Ctrl
Mod4+Ctrl+Shift

Mod1
Mod1+Shift

Mod1+Mod4
Mod1+Mod4+Shift

No modifiers
Shift
Ctrl
Ctrl+Shift
```

(And perhaps some way to filter prefixes (e.g. selecting `Mod4`) and show only those keyboard layouts, to prevent information overload. Or a table of contents with relative links. But that's outside the scope of this issue!)

**Comment by RasmusLindroth** [comment]
Good feedback! This should be an easy fix :)

**Comment by RasmusLindroth** [comment]
I've fixed so it sorts by the group of modifiers with the most modifiers.

I will add a TOC to the web view later, and regarding filtering I need to think a little bit more about it. It's maybe most relevant the text and svg outputs? The web UI can be polished so it's more space between each keyboard so it doesn't look so overwhelming.

**Comment by RasmusLindroth** [comment]
Closed in #11
